tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Hangfire/ICreateContextProxy.cs

```diff
@@ -0,0 +1,33 @@
+// <copyright file="ICreateContextProxy.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+#nullable enable
+
+using Datadog.Trace.DuckTyping;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Hangfire;
+
+/// <summary>
+/// DuckTyping interface for Hangfire.Client.CreateContext
+/// </summary>
+/// <remarks>
+/// https://github.com/HangfireIO/Hangfire/blob/96c5d825ab3ee6f123f9e041ac301881e168e508/src/Hangfire.Core/Client/CreateContext.cs
+/// </remarks>
+internal interface ICreateContextProxy : IDuckType
+{
+    /// <summary>
+    /// Gets a value of System.Collections.Generic.IDictionary`2[System.String,System.Object]
+    /// </summary>
+    object Items { get; }
```

Comment:
Since the types are fully contained in the framework, we can strongly type against the object, instead of just using `object`

```suggestion
    IDictionary<string, object?> Items { get; }
```